rero_ils/modules/acq_orders/api.py

```diff
@@ -261,69 +275,91 @@ def get_related_notes(self, resource_filters=None):
     def get_order_lines(self, output=None, includes=None):
         """Get order lines related to this order.
 
-        :param output: output method. 'count' or None.
+        :param output: output method. 'count', 'query' or None.
         :param includes: a list of statuses to include order lines.
         :return a generator of related order lines (or length).
         """
-        query = AcqOrderLinesSearch()\
-            .filter('term', acq_order__pid=self.pid)
+        query = AcqOrderLinesSearch().filter('term', acq_order__pid=self.pid)
         if includes:
             query = query.filter('terms', status=includes)
 
         if output == 'count':
             return query.count()
-        return self._list_object_by_id(AcqOrderLine, query)
+        elif output == 'query':
+            return query
+        else:
+            return self._list_object_by_id(AcqOrderLine, query)
 
-    def get_order_total_amount(self):
-        """Get total amount of order."""
+    def get_order_encumbrance_total_amount(self):
+        """Get total amount of encumbrance of this order."""
```

Comment:
Should we `exclude` the `RECEIVED` and `PARTIALLY_RECEIVED` as well. Those `statuses` are categorized as `expenditure`